docs/index.md

```diff
@@ -53,7 +53,7 @@ ViewComponent methods are implemented within the scope of the template, encapsul
 
 ### Testing
 
-Unlike traditional Rails templates, ViewComponents can be unit tested. In the GitHub codebase, ViewComponent unit tests are over 100x faster than similar controller tests.
+ViewComponents are meant to be unit tested. In the GitHub codebase, ViewComponent unit tests are over 100x faster than similar controller tests.
```

Comment:
```suggestion
ViewComponent was designed with the intention that all components should be unit tested. In the GitHub codebase, ViewComponent unit tests are over 100x faster than similar controller tests.
```

I think something like this could make a stronger argument of "if you aren't testing your components, you're doing something wrong", but the existing wording is great too. 👍